docs/source/get_started/kedro_concepts.md

```diff
@@ -0,0 +1,74 @@
+# Kedro concepts
+
+It is time to introduce the most basic elements of Kedro.
+
+## Node
+
+In Kedro, a node is a wrapper for a Python function that names the inputs and outputs of that function. Nodes are the building block of a pipeline, and the output of one node can be the input of another.
```

Comment:
I pushed for this, I quite like it

Reply:
"pure" == no side effects, right? So we mean stateless.

I think we should avoid too much complexity at this stage. I do however want to put a graphic on this page that combines the best of these two graphics (I'm still working on a prototype for Mackay)

<img width="756" alt="Screenshot 2022-11-18 at 10 55 49" src="https://user-images.githubusercontent.com/5180475/202760958-40d99ddf-8f69-4d51-88db-8ebf02c03b1d.png">
<img width="757" alt="Screenshot 2022-11-18 at 09 53 36" src="https://user-images.githubusercontent.com/5180475/202760967-0cfa22c2-de1c-4f49-bab1-18420e86693f.png">

Reply:
I like the pure Python function concepts but I also agree that introduce these term earlier could be a bit intimidating.

Reply:
Yeah we could also just hyperlink to this one the word [pure](https://realpython.com/python-functional-programming/#:~:text=A%20pure%20function%20is%20a,to%20state%20or%20mutable%20data.)

Reply:
I think I'll add a definition to the glossary and link to that rather than to the resource, which is great, but is itself a bit intimidating. I'll add the definition to Kedro terms and link to the resource from there, so it's a second order link for the keenest readers only :)